app/models/piece.rb

```diff
@@ -97,5 +114,5 @@ def obstructed?(x, y)
     end
   end
   
-end # end of class
+end
 
```

Comment:
Thanks so much for commenting this whole thing out. It was definitely readable before, but this is so much simpler. :+1:

Reply:
Thanks! I never feel fully done,  so if anything is clear i'm always here.